### ISSUE TITLE
upgrade to 6.5.0 throw error

### ISSUE THREAD

**Issue by hellolovebuxiu** [jsx-eslint/eslint-plugin-jsx-a11y#824]
rule.create is not a function Occurred while linting

**Comment by martinpitt** [comment]
I see this in https://github.com/cockpit-project/cockpit-podman/ and other projects as well now:
```
ESLint: 7.32.0

TypeError: Error while loading rule 'jsx-a11y/alt-text': rule.create is not a function
Occurred while linting /var/home/martin/upstream/cockpit-podman/src/ContainerCheckpointModal.jsx
    at createRuleListeners (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:765:21)
    at /var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:937:31
    at Array.forEach (<anonymous>)
    at runRules (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:882:34)
    at Linter._verifyWithoutProcessors (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:1181:31)
    at Linter._verifyWithConfigArray (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:1280:21)
    at Linter.verify (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:1235:25)
    at Linter.verifyAndFix (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/linter/linter.js:1428:29)
    at verifyText (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/cli-engine/cli-engine.js:240:48)
    at CLIEngine.executeOnFiles (/var/home/martin/upstream/cockpit-podman/node_modules/eslint/lib/cli-engine/cli-engine.js:808:28)
```

**Comment by TomCorvus** [comment]
Hi there,
Same error for me since 6.5.0: 

```
Oops! Something went wrong! :(

ESLint: 8.2.0

TypeError: Error while loading rule 'jsx-a11y/alt-text': rule.create is not a function
```

**Comment by avalenti89** [comment]
same here. It doesn't blocks me on local, but it does in github action container.

I deleted node_modules and packages-lock too

**Comment by ddolcimascolo** [comment]
Same here! It's blocking our CI

**Comment by randombitsUK** [comment]
See a similar issue:
TypeError: Error while loading rule 'jsx-a11y/anchor-has-content': rule.create is not a function

I think it's maybe bacause 6.5.0 switches to support ESlint 8, and we're using 7.32?

**Comment by ljharb** [comment]
Yes thanks, this is why i unmarked it as “latest” last night, an hour or so after releasing it - so most people shouldn’t be prompted to update to it.

Downgrade to 6.4 for now; when I’ve fixed it I’ll publish a 6.5.1 and mark that as latest.
